app/controllers/admin/config/about_controller.rb

```diff
@@ -5,5 +5,44 @@ def index
   end
 
   def update
+    settings_map = {}
+    if general_settings = params[:general_settings]
+      settings_map[:title] = general_settings[:name]
```

Comment:
I think it would be easier on yourself if you made all the keys sent by the client the same as their actual site setting names, then you could do `settings_map.merge!(general_settings)` and so on.

Reply:
There are a couple of reasons why I went with the current approach:

1. I didn't want to make it possible to change arbitrary site settings via this API endpoint. If I merged the `params` object directly into the `settings_map`, then it would be possible for a client to shove in an unrelated site setting and the change would go through. It wouldn't be a security issue, but feels unhygienic.

2. Some setting names don't match their field names in the UI because we're reusing existing settings, but I think it will be nice to have the params for the API endpoint match the names of the fields in the UI.

Reply:
Okay yes I see your reasoning now. Let's leave it as is for now then, maybe later on as we do more of these mixed setting config pages we can think of some sensible pattern to take bulk keys/values from the UI and convert them to a big hash of only allowed settings to update for a certain endpoint :+1: